ManagementSystem/PhoneBook.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const PersonalContact = require('./PersonalContact');
+const BusinessContact = require('./BusinessContact');
+const { ContactNotFoundError, FileSystemError } = require('./Errors');
+
+const fileName = 'contacts.txt';
+
+class PhoneBook {
+    constructor() {
+        this.contacts = [];
+        this.loadContactsFromFile();
+    }
+
+    addContact(contact) {
+        this.contacts.push(contact);
```

Comment:
Where are you validating that ContactAlreadyExists or any duplicity?

Reply:
In app.js file I used promptForUniqueName function which is created in UserInputs.js file. This function will make sure the contact name is unique.